crates/sui-sdk/examples/transfer_coins.rs

```diff
@@ -4,14 +4,10 @@
 use shared_crypto::intent::Intent;
 use sui_json_rpc_types::SuiTransactionResponseOptions;
 use sui_keys::keystore::{AccountKeystore, FileBasedKeystore, Keystore};
-use sui_sdk::{
-    types::{
-        base_types::{ObjectID, SuiAddress},
-        messages::Transaction,
-    },
-    SuiClientBuilder,
-};
+use sui_sdk::SuiClientBuilder;
```

Comment:
question: what's the diff between sui_sdk::types and sui_types?

Reply:
sui_types are basic sui protocol types, sui-json-rpc-types are user facing ones, we did not have sdk types until this one newly added one, changes are needed here b/c of types overriding.